ova/Config_files/filebeat.yml

```diff
@@ -0,0 +1,58 @@
+# Wazuh - Filebeat configuration file
```

Comment:
This file isn't necessary. This file can be downloaded from https://github.com/wazuh/wazuh/blob/3.9/extensions/filebeat/7.x/filebeat.yml.